supertokens_python/recipe/emailverification/types.py

```diff
@@ -12,7 +12,6 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-
```

Comment:
Not sure why this change is here.

Reply:
I think this is happening due to autopep8. It will be fixed when we use `black`.